packages/math-input/src/components/popover-state-machine.ts

```diff
@@ -104,8 +123,8 @@ class PopoverStateMachine {
             this.activePopover = id;
             this.handlers.onActiveNodesChanged({
                 popover: {
-                    parentId: this.activePopover,
```

Comment:
Logic change: the type of activePopover can be `null`, but we literally just set it so it's not. Instead I'm just using the value we used to set activePopover.